src/Symfony/Component/ExpressionLanguage/Tests/ExpressionLanguageTest.php

```diff
@@ -311,12 +311,25 @@ public function testNullSafeEvaluateFails($expression, $foo, $message)
     /**
      * @dataProvider provideInvalidNullSafe
      */
-    public function testNullSafeCompileFails($expression, $foo)
+    public function testNullSafeCompileFails($expression)
```

Comment:
`$foo` should not be removed. It is used by the eval'd code.

Reply:
see #49391